packages/babel-compat-data/scripts/download-compat-table.sh

```diff
@@ -16,5 +16,5 @@ fi
 
 rm -rf build/compat-table
 mkdir -p build
-git clone --branch=gh-pages --single-branch --shallow-since=2020-04-01 https://github.com/kangax/compat-table.git build/compat-table
-cd build/compat-table && git checkout -qf $COMPAT_TABLE_COMMIT
+git clone --single-branch --shallow-since=2020-04-01 https://github.com/kangax/compat-table.git build/compat-table
+cd build/compat-table && git checkout -q $COMPAT_TABLE_COMMIT
```

Comment:
`-f` is to suppress warning when working tree is different than HEAD. This is highly unlikely since we have removed `build/compat-table` before cloning.